src/storybook/stories/Section.stories.tsx

```diff
@@ -0,0 +1,194 @@
+import React from 'react';
+import {
+  ActionSectionItem,
+  ButtonSectionItem,
+  CounterSectionItem,
+  DateInputSectionItem,
+  ExpandableSectionItem,
+  FavoriteDepartureSectionItem,
+  FavoriteSectionItem,
+  GenericClickableSectionItem,
+  GenericSectionItem,
+  HeaderSectionItem,
+  LinkSectionItem,
+  LocationInputSectionItem,
+  MessageSectionItem,
+  PhoneInputSectionItem,
+  Section,
+  SectionProps,
+  TextInputSectionItem,
+  TimeInputSectionItem,
+  ToggleSectionItem,
+  InternalLabeledSectionItem,
+  RadioGroupSection,
+  ContainerSizingType,
+} from '@atb/components/sections';
+import {Meta} from '@storybook/react-native';
+import {ScrollView, View} from 'react-native';
+import {
+  themedStoryControls,
+  ThemedStoryDecorator,
+  ThemedStoryProps,
+} from '@atb/storybook/ThemedStoryDecorator';
+import {ThemeText} from '@atb/components/text';
+
+type SectionMetaProps = SectionProps & ThemedStoryProps;
+const containerSizingType: ContainerSizingType[] = [
+  'inline',
+  'block',
+  'compact',
+  'spacious',
+];
+
+const SectionMeta: Meta<SectionMetaProps> = {
+  title: 'Section',
+  component: Section,
+  argTypes: {
+    ...themedStoryControls,
+    withPadding: {control: 'boolean'},
+    withFullPadding: {control: 'boolean'},
+    withTopPadding: {control: 'boolean'},
+    withBottomPadding: {control: 'boolean'},
+    type: {
+      control: 'select',
+      options: containerSizingType,
+    },
+  },
+  args: {
+    theme: 'light',
```

Comment:
I guess it can be a good idea to use `...themedStoryDefaultArgs,` here, as it is used for all others. If new default args are added, they should probably be included here as well.